Dockerfile.dapper

```diff
@@ -10,14 +10,15 @@ RUN zypper -n install gcc ca-certificates git wget curl vim less file python3-to
     rm -rf /var/cache/zypp/*
 
 RUN if [ ${ARCH} == "s390x" ]; then \
-         ln -s /usr/bin/gcc /usr/bin/s390x-linux-gnu-gcc;\
-     fi
+    ln -s /usr/bin/gcc /usr/bin/s390x-linux-gnu-gcc;\
+    fi
 
 ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm64=arm64 GOLANG_ARCH_s390x=s390x GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
     GOPATH=/go PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
 
-RUN wget -O - https://storage.googleapis.com/golang/go1.21.3.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
-    go install golang.org/x/lint/golint@latest
+RUN wget -O - https://storage.googleapis.com/golang/go1.21.3.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local
+
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.54.2
```

Comment:
the latest version is ` v1.55.2`, better to use it instead?

Reply:
Updated it.